API/Controllers/ProjectController.cs

```diff
@@ -206,6 +225,24 @@ public async Task<IActionResult> UpdateProject(int projectId, [FromBody] Project
             }
 
             mapper.Map(projectResource, project);
+            File file = null;
+            if(projectResource.FileId != 0)
+            {
+                file = await fileService.FindAsync(projectResource.FileId);
```

Comment:
```suggestion
                file = await fileService.FindAsync(projectResource.FileId);
                project.FileId = file.Id
```

Reply:
I think this is needed right? In case the file is updated, you want to update the file id of the project as well

Reply:
It's automatically updated after saving the project.
Just checked it. Just updated id 1127: https://gyazo.com/50800f41fc637714c3a13e99003a378a